RemoveGetterSetter/ImmutableObjects/Person.java

```diff
@@ -0,0 +1,11 @@
+package RemoveGetterSetter.ImmutableObjects;
+
+public class Person {
+    public final String name;
```

Comment:
Any particular reason why we are using final here, if I dont use final this will not work?